src/js/common/schemaform/state/reducers.js

```diff
@@ -12,6 +15,23 @@ import {
 } from '../state/helpers';
 
 export default {
+  [OPEN_REVIEW_CHAPTER]: (state, action) => {
+    const openChapters = _.concat(
+      _.get('reviewPageView.openChapters', state) || [],
```

Comment:
We could probably default this to [] in the initial state.

Reply:
Good idea.